src/static/css/content.css

```diff
@@ -50,3 +50,9 @@
         display: none !important;
     }
 }
+
+.TridactylSearchHighlight {
+    opacity: 0.5 !important;
```

Comment:
This is a really ugly hack. Nice work ;)

Reply:
Is it? I don't remember what any of this does but it doesn't seem too shocking right now.

Also, don't bother reading any of this code. I tried to rebase it and it was awful, I should probably just rewrite everything and copy/paste where I can.
Sorry it's taking so long.

Reply:
Oh, I think I see what you mean. This is actually the recommended way of doing this on MDN.

Reply:
I merged master into it because rebases are for masochists. It was fine. : /